cmd/rook/ceph/operator.go

```diff
@@ -43,9 +42,6 @@ https://github.com/rook/rook`,
 }
 
 func init() {
-	operatorCmd.Flags().DurationVar(&mon.HealthCheckInterval, "mon-healthcheck-interval", mon.HealthCheckInterval, "mon health check interval (duration)")
```

Comment:
Are we also not using the `mon.HealthCheckInterval` and `mon.MonOutTimeout`? We should be reading these from the cluster CR now instead of setting package variables. Actually, we likely need to keep those for backward compatibility.